backend/members/lookup.go

```diff
@@ -27,3 +30,134 @@ func (c *Client) RetrieveByEmails(ctx context.Context, emails []string) (*[]Memb
 
 	return get[[]Member](ctx, c, fmt.Sprintf("persons?filter=%s", encoded))
 }
+
+type listOfMembersByIDFilter struct {
+	PersonID map[string]any `json:"personID"`
+}
+
+// RetrieveByID retrieves a batch of members by ID
+func (c *Client) RetrieveByID(ctx context.Context, ids []int) (*[]Member, error) {
+	// TODO: We should likely allow passing any number of ids in here and chunk them internally
+
+	filter := listOfMembersByIDFilter{
+		PersonID: map[string]any{
+			"_in": ids,
+		},
+	}
+
+	encoded, _ := json.Marshal(filter)
+
+	return get[[]Member](ctx, c, fmt.Sprintf("persons?limit=999&fields=personID,age,affiliations.*&filter=%s", encoded))
+}
+
+// CountMembersByAge counts active members who are min <= age <= max
+func (c *Client) CountMembersByAge(ctx context.Context, min, max int) (int, error) {
+	/// TOOO: This is currenty only used by `statistician` and needs to be cleaned up, possibly limited?
+	filter := gin.H{
```

Comment:
Instead of using a package alias for map[string]any, mb create a local/internal shorthand for it? Or does it have some tooling you use? Kind of weird to reference external packages for simple things like these, but that's my thoughts.